content/position-listing/index.md

```diff
@@ -37,6 +37,8 @@ banner:
 <!---- - [Forest Ecology Field Crew Leader (UC student)](/positions/field-crew-leader-ucstudent/) --->
 <!-- - [Summer field technician (UC student)](/positions/field-crew-member/) --->
 <!-- - [Open-Source Software Engineering Intern (UC Student)](/positions/software-engineering-intern/) -->
+- [Data Infrastructure Engineering Intern (UC Student)](/positions/data-infrastructure-engineering-intern/)
```

Comment:
Consider changing to "cloud" infrastructure... since it might be more relevant to people who don't know about our work. Data feels somewhat implicit in most applications.